### ISSUE TITLE
Repartition disks on Fedora Copr hypervisors

### ISSUE THREAD

**Issue by praiskup** [fedora-copr/copr#2869]
This is likely blocking power9 builds

**Comment by praiskup** [comment]
Now root@vmhost-p08-copr01.rdu-cc.fedoraproject.org is down (not SSH responding).

**Comment by praiskup** [comment]
Requested rights to power-cycle: https://pagure.io/fedora-infrastructure/issue/11476

**Comment by praiskup** [comment]
After another reboot, it started working.

**Comment by praiskup** [comment]
Again both p08 machines down :-( being resolved still in the same infra ticket but reopening to "admit" there still is some issue.

**Comment by nikromen** [comment]
and they were down once again

**Comment by praiskup** [comment]
We want to move `/` partition onto it's own raid, and stop using the `raid6` for swap and disposable epemeral images (raid0 or lvm striping should be used for that).

Tracking progress:

- [ ] vmhost-x86-copr01.rdu-cc.fedoraproject.org
- [ ] vmhost-x86-copr02.rdu-cc.fedoraproject.org
- [ ] vmhost-x86-copr03.rdu-cc.fedoraproject.org
- [ ] vmhost-x86-copr04.rdu-cc.fedoraproject.org
- [ ] vmhost-p08-copr01.rdu-cc.fedoraproject.org
- [ ] vmhost-p08-copr02.rdu-cc.fedoraproject.org
- [ ] vmhost-p09-copr01.rdu-cc.fedoraproject.org

**Comment by praiskup** [comment]
https://nagios.fedoraproject.org/nagios/cgi-bin//status.cgi?hostgroup=copr_hypervisor&style=detail

**Comment by praiskup** [comment]
The (rather long, eventually) post is here: https://pavel.raiskup.cz/blog/fedora-copr-hypervisor-disk-repartitioning.html

**Comment by praiskup** [comment]
I spent at least an hour writing a good report now against LibVirt, and then
I realized that actually, the Power9 machine on Fedora 38 doesn't suffer from the
same qcow2 issues (it is on SSD, but still).  The other machines are slow, but also
very old.  While it doesn't make sense to have a disk slowdown with qcow2 after
`rhel6` to `rhel0` to me, I don't think I want to bother LibVirt folks (OK on newer
hardware, OK also on older after moving to RAW).  Giving up the last sub-task.
